packages/app-store/routing-forms/config.json

```diff
@@ -2,17 +2,19 @@
   "/*": "Don't modify slug - If required, do it using cli edit command",
   "name": "Routing Forms",
   "title": "Routing Forms",
+  "isGlobal": true,
   "slug": "routing-forms",
   "type": "routing-forms_other",
   "logo": "icon-dark.svg",
   "url": "https://cal.com/resources/feature/routing-forms",
   "variant": "other",
   "categories": ["automation"],
   "publisher": "Cal.com, Inc.",
+  "simplePath": "/routing-forms",
```

Comment:
See App.d.ts for definition

Reply:
Would that mean that this URL should work? Or am I misunderstanding this? 
![Screenshot 2023-07-26 at 12 16 48](https://github.com/calcom/cal.com/assets/30310907/4be1f0ea-6234-4c51-921a-ae1654da8f39)

Reply:
Agreed with @CarinaWolli it's kinda confusing as I also expected that URL to work. But it only works when you go to `/routing-forms/forms`. Not a blocker IMO.

Reply:
Fixed it here https://github.com/calcom/cal.com/pull/10412. Certainly a bug.